.goreleaser-template.yaml

```diff
@@ -868,8 +871,15 @@ release:
   github:
     owner: openbao
     name: openbao
-  draft: true #${{ .Env.GITHUB_RELEASE_DRAFT }}
-  replace_existing_draft: false
+
   prerelease: ${{ .Env.GITHUB_PRERELEASE }}
   make_latest: ${{ .Env.GITHUB_RELEASE_MAKE_LATEST }}
   disable: false
+
+  # We overwrite the following variables to support multi-stage execution of
+  # goreleaser on the same artifacts with different GOOS values, to avoid
+  # running out of disk space. We can't support draft releases because the same
+  # release needs to be used for all artifacts uploaded from separate stages.
+  draft: false
+  replace_existing_draft: false
+  replace_existing_artifacts: false
```

Comment:
Not sure if we want this to be `false` or `true` tbh; my worry is that separate matrix jobs _might_ have the same artifacts and so we'd be overwriting them, but we might also want to support re-executing the entire release pipeline if something failed somewhere.

Thoughts?

Reply:
I think it should be fine to overwrite any existing artifacts, but let's try without first